### DISCUSSION TITLE
Log the header when the header size exceeded to debug problems easily

### DISCUSSION THREAD

**Discussion by irshadcc** [netty/netty#14190] [General]
The current HpackEncoder doesn't show the values of headers when the size get's exceeded. Can Netty framework log it so that it is easy to debug ? 

https://github.com/netty/netty/blob/4.1/codec-http2/src/main/java/io/netty/handler/codec/http2/HpackEncoder.java

**Comment by normanmaurer** [comment]
@irshadcc I think adding the headers etc would not be a good idea as it might contain sensitive informations etc.

**Reply by irshadcc** [reply]
Okay
